main.py

```diff
@@ -43,13 +43,15 @@
 from core.domain import user_services
 from core.platform import models
 import feconf
-# pylint: enable=relative-import
 
 from mapreduce import main as mapreduce_main
 from mapreduce import parameters as mapreduce_parameters
 import webapp2
 from webapp2_extras.routes import RedirectRoute
 
+# pylint: enable=relative-import
```

Comment:
Check whether this is still needed, please.

Ditto elsewhere. Make sure to manually audit all files and drop pragmas that are no longer needed.

Reply:
Sure! I will ensure this.

Reply:
Why is this still here? Ditto elsewhere.

It used to exist because of mapreduce/webapp2 I think, but if that's not needed any more then I'm not sure what it's point is. Could you clarify?

Also, make sure to run the app locally (plus at least one backend test) to ensure all of the infrastructure still works after your changes.

Reply:
I tried removing it and then running the lint checks. It produces an error as Pylint recommends `import .feconf` instead of `import feconf`. It is not because of mapreduce/webapp2.

Reply:
Ah ok. I'm fine with `import .feconf`, though it's a bit unfamiliar. Does `from . import feconf` work? If so, that would probably be better.

Also, if we change it here, we should change it everywhere, for consistency.

Reply:
Okay, so this is the error produced in `main.py`:  
![import-patch](https://user-images.githubusercontent.com/24826041/34559743-02200ce2-f169-11e7-8510-1517a78fc3d9.png)  
I think it will difficult to do away with `pylint: enable=relative-import` as we will have to change it manually for each such import.  
Please suggest what should be done. We can also disable relative import check from .pylintrc and then remove these statements.    
Thanks!

Reply:
I see, interesting. OK, let's go with the enable-relative-import pragma then, as we had it before.